Unix/common/MI.h

```diff
@@ -8500,6 +8508,138 @@ MI_INLINE MI_Result MI_DestinationOptions_GetDestinationPort(
     }
 }
 
+
+/*
+**=============================================================================
+**
+** MI_DestinationOptions_SetTrustedCertsDir()
+**
+**=============================================================================
+*/
+MI_INLINE MI_Result MI_DestinationOptions_SetTrustedCertsDir(
+    _Inout_   MI_DestinationOptions *options,
+    _In_z_ const MI_Char *certsdir)
+{
+    if (options && options->ft)
```

Comment:
Since you're checking parameters, may as well check certsdir too.

if (options && options->ft && certsdir)